pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -62,6 +72,7 @@ QubitSum               = {}
 SISWAP                 = {}
 SQISW                  = {}
 SX                     = {}
+MultiControlledX       = {}
```

Comment:
This is natively (C++) supported in Lightning

Reply:
It was previously in `operators.gates.decomp`, which means that Catalyst was explicitly told to decompose this. Do we want to change that?

Reply:
Hmm.. okay -- there isn't any frontend support in Catalyst for MCX. It should be easy to fix as a follow-up.